CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to Rust's notion of
 
 ### Changed
 - Add `repr(transparent)` attribute to Fp/Fq structures.
+- Omit 'infinity' field from affine coordinates structures and use (0, 0)
+  to denote the identity points.
```

Comment:
I don't think this needed to be documented as it was a private implementation detail, and the implementation details of curve points were never implicitly exposed before the new `repr-c` feature flag. But no harm in doing so.